src/router/router.ts

```diff
@@ -1,24 +1,34 @@
-import { createRouter as createVueRouter, createWebHistory, NavigationGuard, Router, RouteRecordRaw } from 'vue-router'
+import {
+  createRouter as createVueRouter,
+  createWebHistory,
+  NavigationGuard,
+  Router,
+  RouteRecordRaw,
+  RouteLocation,
+  RouteLocationRaw,
+} from 'vue-router'
 
 import { getLastNumberParameter } from './getLastParameter'
 import { store } from '@/store/store'
 import { PolicyDefinition } from '@/types/index.d'
 import { ClientStorage } from '@/utilities/ClientStorage'
 
 function getPolicyRoutes(policies: PolicyDefinition[]): RouteRecordRaw[] {
-  return policies.map((policy) => ({
-    path: policy.path,
-    name: policy.path,
-    meta: {
-      shouldReRender: true,
-      title: policy.pluralDisplayName,
-    },
-    props: (route) => ({
-      policyPath: policy.path,
-      offset: getLastNumberParameter(route.query.offset),
-    }),
-    component: () => import('@/app/policies/views/PolicyView.vue'),
-  }))
+  return policies.map((policy) => {
+    return {
+      path: policy.path,
+      name: policy.path,
+      meta: {
+        shouldReRender: true,
+        title: policy.pluralDisplayName,
```

Comment:
**Issue**: Currently, the sidebar item is not highlighted which can be fixed by marking the `parent` in its meta properties. We use this field for things like this and as part of the breadcrums.

```suggestion
        title: policy.pluralDisplayName,
        parent: 'policies',
```